lib/spaceship/tunes/tunes_client.rb

```diff
@@ -745,8 +756,8 @@ def create_tester!(tester: nil, email: nil, first_name: nil, last_name: nil)
         req.headers['Content-Type'] = 'application/json'
       end
 
-      data = parse_response(r, 'data')['testers']
-      handle_itc_response(data) || data[0]
+      data = parse_response(r, 'data')[arr_key]
+      handle_itc_response(data) || data.detect { |element| element['emailAddress']['value'] == email }
```

Comment:
I don' think it is totally necessary here, this code is finding the element that we just added, so the strings will be identical.  This particular change is required because when adding a tester by app, the response is the list of testers for the app rather than just the single tester that was created (data[0]).